incubator/distributed-tensorflow/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for distributed-tensorflow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+worker:
+  number: 2
+  podManagementPolicy: Parallel
+  image:
+    repository: cheyang/distributed-tf
```

Comment:
Is there an official image?

Reply:
Here is the docker hub info: https://hub.docker.com/r/cheyang/distributed-tf/